backend/src/app.ts

```diff
@@ -75,6 +75,11 @@ server
 server.route('/itemComplete').post(items.itemComplete);
```

Comment:
I don't feel these changes should be in this PR, did you merge Ronalds branch by chance?